src/System.Threading.Tasks.Extensions/tests/AsyncValueTaskMethodBuilderTests.cs

```diff
@@ -83,36 +83,6 @@ public void Start_InvokesMoveNext()
         }
 
         [Theory]
-        [InlineData(false)]
-        [InlineData(true)]
-        public async Task AwaitOnCompleted_InvokesStateMachineMethods(bool awaitUnsafe)
-        {
-            AsyncValueTaskMethodBuilder<int> b = ValueTask<int>.CreateAsyncMethodBuilder();
-            var ignored = b.Task;
-
-            var callbackCompleted = new TaskCompletionSource<bool>();
-            IAsyncStateMachine foundSm = null;
-            var dsm = new DelegateStateMachine
-            {
-                MoveNextDelegate = () => callbackCompleted.SetResult(true),
-                SetStateMachineDelegate = sm => foundSm = sm
-            };
-
-            TaskAwaiter t = Task.CompletedTask.GetAwaiter();
-            if (awaitUnsafe)
-            {
-                b.AwaitUnsafeOnCompleted(ref t, ref dsm);
-            }
-            else
-            {
-                b.AwaitOnCompleted(ref t, ref dsm);
-            }
-
-            await callbackCompleted.Task;
-            Assert.Equal(dsm, foundSm);
-        }
```

Comment:
Looks like you'll need to delete https://github.com/adityamandaleeka/corefx/blob/ae7f6350baa5798459936a00e3b24e0e05356e92/src/System.Threading.Tasks.Extensions/tests/AsyncValueTaskMethodBuilderTests.cs#L187 as well, or else you'll get warnings-as-errors for not using that.